packages/styles/scss/components/card/index.scss

```diff
@@ -133,14 +134,27 @@
       }
 
       ::slotted(#{$dds-prefix}-tag-group) {
-        margin-bottom: 1rem;
+        margin-bottom: $spacing-05;
       }
     }
     &[color-scheme='light'] {
       &,
       &:hover {
         .#{$prefix}--card__wrapper {
           background-color: $ui-02;
+          transition: $duration--moderate-01 motion(standard, productive);
```

Comment:
Am I seeing this right? Looks like at least on the default color scheme the transition only applies to on mouse out and not mouse in.

![May-27-2021 10-49-11](https://user-images.githubusercontent.com/3793636/119857460-33bae880-bed9-11eb-9064-49d84e699368.gif)

Reply:
Needed to add the transition property to the wrapper class as well, pushed the changes 👍